app/views/posts/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1> All Scribble Posts</h1>
+
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Title</th>
+    <th>Content</th>
+    <th></th>
+    <th></th>
+    <th></th>
+  </tr>
+
+<% @posts.each do |post| %>
+  <tr>
+    <td><%= post.name %></td>
+    <td><%= link_to post.title, post_path(post) %></td>
+    <td><%= post.content %></td>
+    <td><%= link_to 'Edit', edit_post_path(post) %></td>
+    <td><%= link_to 'Delete', post, :confirm => 'Are you sure?',
+                                     :method => :delete %></td>
+  </tr>
+<% end %>
```

Comment:
tables are generally for tabular data. Try using divs and p tags other other layouts for your models. Start thinking about how you can make this your own.